Makefile.win

```diff
@@ -4,12 +4,16 @@ MIX_APP_PATH=$(MAKEDIR)
 
 PRIV_DIR = $(MIX_APP_PATH)/priv
 NIF_SO = $(PRIV_DIR)/adbc_nif.dll
-ADBC_SRC = $(MAKEDIR)\3rd_party\apache-arrow-adbc
+THIRD_PARTY_DIR = $(MAKEDIR)\3rd_party
+ADBC_SRC = $(THIRD_PARTY_DIR)\apache-arrow-adbc
 ADBC_C_SRC = $(ADBC_SRC)\c
 C_SRC = $(MAKEDIR)\c_src
 !IFDEF CMAKE_TOOLCHAIN_FILE
 CMAKE_CONFIGURE_FLAGS=-D CMAKE_TOOLCHAIN_FILE="$(CMAKE_TOOLCHAIN_FILE)"
 !ENDIF
+!IFDEF ABDC_DRIVER_SQLITE
+ABDC_DRIVER_SQLITE=true
+!ENDIF
```

Comment:
Same here?

Reply:
Oops, sorry I missed them. Will remove it now :)